emails/tests/views_tests.py

```diff
@@ -351,6 +354,83 @@ def test_noreply_email_in_s3_deleted(self, mocked_message_removed):
         assert response.status_code == 200
         assert response.content == b"noreply address is not supported."
 
+    @override_settings(STATSD_ENABLED=True)
+    @patch("emails.views.remove_message_from_s3")
+    @patch("emails.views._get_keys_from_headers")
+    def test_no_header_reply_email_in_s3_deleted(
+        self, mocked_get_keys, mocked_message_removed
+    ):
+        """If replies@... email has no "In-Reply-To" header, delete email, return 400."""
+        mocked_get_keys.side_effect = InReplyToNotFound()
+
+        with MetricsMock() as mm:
+            response = _sns_notification(EMAIL_SNS_BODIES["s3_stored_replies"])
+        mm.assert_incr_once(
+            "fx.private.relay.reply_email_header_error", tags=["detail:no-header"]
+        )
```

Comment:
praise: So glad we are using MetricsMock to test that increments work as expected! Will use this in the future, thanks for adding it.